packages/integrations/stores/launch-darkly/README.md

```diff
@@ -0,0 +1,45 @@
+# @configu-integrations/launch-darkly
+
+Integrates the Configu Orchestrator with [LaunchDarkly](https://launchdarkly.com/).
+
+- Name: LaunchDarkly
+- Category: Feature flag manager
+
+## Configuration
+
+Configu needs to be authorized to access your LaunchDarkly project. You must specify an `sdkKey` that corresponds to an SDK key of an environment of some LaunchDarkly project and a context that has a `key` or a `targetingKey` that identifies the subject (end-user, or client service) of a flag evaluation. You may also configure [`ldOptions`](https://launchdarkly.github.io/node-server-sdk/interfaces/_launchdarkly_node_server_sdk_.LDOptions.html) to customize the LaunchDarkly client.
+
+## Usage
+
+### `.configu` store declaration
+
+```yaml
+stores:
+  my-store:
+    type: launch-darkly
+    configuration:
+      sdkKey: example-sdkKey
+        context:
```

Comment:
is that the correct indentation?
it seems like "context" and "sdkKey" should be siblings.

Reply:
It's correct, that's how the underlying SDK expects to get the parameter